seanjhulse/sherlock
# Move socket data to background process

This changes our **/network-traffic/** URL to provide an example websocket connection for receiving TCP and UDP data

How to test?

- Start the webserver
- Go to http://localhost:8000/websockets/

You can look at the JavaScript to see how to connect to it yourself.
